### PR TITLE
Add marine human pressures layers

### PR DESCRIPTION
**Pull request by weberjavi**
### Description
This PR adds marine pressures layers to the global sidebar on the data globe
### Testing instructions
Play with the human pressures layers on the global sidebar.
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-288

### CODE REVIEW

**Comment by gretacv** [comment]
Excellent Javi!
Small thing: in the side bar Commercial fishing is not capitalised.
Maybe @tamaramegan can you check that the tiles are well linked (in case we mixed up the names or similar)?

**Comment by tamaramegan** [comment]
The tiles look fine(the categories match the names), but I have realised that I need to expand the tiles to more zoom levels. Will add a task in JIRA for that.

**Comment by weberjavi** [comment]
> Excellent Javi!
> Small thing: in the side bar Commercial fishing is not capitalised.

@gretacv do you feel like making a suggestion that could be commited right away?
[Check this article ](https://haacked.com/archive/2019/06/03/suggested-changes/)in case you do not know how to do it
:octocat:

File: src/constants/human-pressures.js [hunk 1]
```diff
@@ -21,14 +24,20 @@ const RAINFED = 'Rainfed agriculture';
 const IRRIGATED = 'Irrigated agriculture';
 const URBAN = 'Urban pressures';
 const RANGELAND = 'Rangeland';
-const MARINE = 'Marine human modification';
+const LAND_DRIVERS = 'Marine land drivers';
+const OCEAN_DRIVERS = 'Marine ocean drivers';
+const COMMERCIAL_FISHING = 'commercial fishing';
```
Comment (gretacv) [inline src/constants/human-pressures.js:29]: const COMMERCIAL_FISHING = 'Commercial fishing';

Comment (gretacv) [inline src/constants/human-pressures.js:29]:
```suggestion
const COMMERCIAL_FISHING = 'Commercial fishing';
```

**Comment by weberjavi** [comment]
@gretacv do you approve this PR?

**Review by gretacv — APPROVED** [review]
Passed my review! 🚀

**Pull request merged**